pkg/scheduler/metrics/metrics.go

```diff
@@ -76,6 +91,14 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
 		},
 	)
+	BindingLatency = prometheus.NewHistogram(
```

Comment:
Looks like this metric is now deprecated and replaced by `Binding`. Could you please add a comment to mark it deprecated?

Reply:
Imo, this metric shouldn't be deprecated - histogram metric provides different data than summary metric. I think both metrics are useful and could be used to measure different aspects.